pkg/controller/v1alpha1/studyjob/studyjob_controller.go

```diff
@@ -542,21 +542,6 @@ func (r *ReconcileStudyJobController) checkStatus(instance *katibv1alpha1.StudyJ
 }
 
 func (r *ReconcileStudyJobController) getAndRunSuggestion(instance *katibv1alpha1.StudyJob, c katibapi.ManagerClient, ns string) (bool, error) {
-	//Check Suggestion Count
-	sps, err := getSuggestionParam(c, instance.Status.SuggestionParameterID)
-	if err != nil {
-		return false, err
-	}
-	for i := range sps {
-		if sps[i].Name == "SuggestionCount" {
-			count, _ := strconv.Atoi(sps[i].Value)
-			if count >= instance.Status.SuggestionCount+1 {
-				//Suggestion count mismatched. May be duplicate suggestion request
-				return false, nil
-			}
-			sps[i].Value = strconv.Itoa(instance.Status.SuggestionCount + 1)
-		}
-	}
```

Comment:
in getSuggestion(line 561), hyperband service will update suggestionParam and store it into DB. If we use `sps` cache from line 546 to update suggestionParam line 614 in DB, it will overwrite values hyperband service updated. That's why hyperband service cannot move on